test/e2e/network/ingress.go

```diff
@@ -416,6 +417,53 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			if pollErr != nil {
 				framework.ExpectNoError(fmt.Errorf("Timed out waiting for ingress %s to get %s annotation", name, instanceGroupAnnotation))
 			}
+
+			// Verify that the ingress does not get other annotations like url-map, target-proxy, backends, etc.
+			// Note: All resources except the firewall rule have an annotation.
+			umKey := fmt.Sprintf("%v/url-map", statusPrefix)
+			fwKey := fmt.Sprintf("%v/forwarding-rule", statusPrefix)
+			tpKey := fmt.Sprintf("%v/target-proxy", statusPrefix)
+			fwsKey := fmt.Sprintf("%v/https-forwarding-rule", statusPrefix)
+			tpsKey := fmt.Sprintf("%v/https-target-proxy", statusPrefix)
+			scKey := fmt.Sprintf("%v/ssl-cert", statusPrefix)
+			beKey := fmt.Sprintf("%v/backends", statusPrefix)
+			wait.Poll(2*time.Second, time.Minute, func() (bool, error) {
+				ing, err := f.ClientSet.ExtensionsV1beta1().Ingresses(ns).Get(name, metav1.GetOptions{})
+				framework.ExpectNoError(err)
+				annotations := ing.Annotations
+				if annotations != nil && (annotations[umKey] != "" || annotations[fwKey] != "" || annotations[tpKey] != "" || annotations[fwsKey] != "" || annotations[tpsKey] != "" || annotations[scKey] != "" || annotations[beKey] != "") {
```

Comment:
nit: could we make the condition multi-line?

Reply:
done